src/System.Drawing.Common/src/misc/ClientUtils.cs

```diff
@@ -124,15 +124,15 @@ public static bool IsEnumValid_NotSequential(System.Enum enumValue, int value, p
 
 #if DEBUG      
         [ThreadStatic]
-        private static Hashtable t_enumValueInfo;
-        public const int MAXCACHE = 300;  // we think we're going to get O(100) of these, put in a tripwire if it gets larger.
+        private static Hashtable s_enumValueInfo;
```

Comment:
Thanks. We do specify that ThreadStatic fields should be prefixed with `t_` in [this doc], item 3. These things are extremely rare, though. I wouldn't be surprised if we only have a handful in the BCL.